### DISCUSSION TITLE
WinXP install for 1.1.6

### DISCUSSION THREAD

**Discussion by AlanHK** [enzo1982/freac#447] [Help & Support] [answered]
![shlwapi](https://user-images.githubusercontent.com/12799423/219986608-31073698-a54f-4c10-90db-b885da6f5fcf.png)
I'm running WinXP and installed fre:ac 1.1.6 from the zip:
x86 (32 bit): freac-1.1.6-windows-i686.zip 
-- overwriting my install of 1.1.5.

On launching there is an error message about shlwapi.dll, as attached.
However, if I close that it seemed to run OK.

I reinstalled using the exe:
x86 (32 bit): freac-1.1.6-windows-i686.exe 

Now it runs with no error messages.
So I assume there is some setting the installer does on top of copying files.

**Comment by enzo1982** [comment, marked as answer]
That's a bug that unfortunately slipped into the 1.1.6 release. It affects the online updater component only, so you can still use fre:ac normally.

When you use the installer and install without the online updater component, there will be no error when starting fre:ac.

The issue will be fixed with the upcoming fre:ac 1.1.7 update.

**Comment by AlanHK** [comment]
OK, I deselected the updater when using the installer.  Thanks.
